synapseclient/table.py

```diff
@@ -692,8 +693,65 @@ def __init__(self, **kwargs):
     def postURI(self):
         return '/column'
 
+@six.add_metaclass(ABCMeta)
+class AppendableRowset(DictObject):
+    """
+    Abstract Base Class for Rowset and PartialRowset
+    """
+    @abstractmethod
+    def __init__(self, schema, **kwargs):
+        if ('tableId' not in kwargs) and schema:
+            kwargs['tableId'] = id_of(schema)
+
+        if not kwargs.get('tableId',None):
+            raise ValueError("Table schema ID must be defined to create a %s" % type(self).__name__)
+        super(AppendableRowset, self).__init__(kwargs)
+
+
+    def _synapse_store(self, syn):
+        """
+        Creates and POSTs an AppendableRowSetRequest_
+
+        .. AppendableRowSetRequest: http://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/AppendableRowSetRequest.html
+        """
+        print(self)
+        append_rowset_request ={'concreteType': concrete_types.APPENDABLE_ROWSET_REQUEST,
+               'toAppend':self,
+               'entityId':self.tableId}
+
+        response = syn._POST_table_transaction(self.tableId, append_rowset_request)
+        return response['results'][0]
+
 
-class RowSet(DictObject):
+
+class PartialRowset(AppendableRowset):
+    """
+    A set of Partial Rows used for Partially updating cells within a table
+
```

Comment:
Should probably be described in more detail.  What does partial rows mean? Or how about "Partially updating cells"